release-notes/4.7.26.md

```diff
@@ -0,0 +1,361 @@
+# CiviCRM 4.7.27
```

Comment:
@agh1 file name looks wrong

Reply:
Thanks @seamuslee001 I had some file name mayhem last night.  Fixed it now.